lib/ansible/module_utils/facts/system/date_time.py

```diff
@@ -37,7 +37,7 @@ def collect(self, module=None, collected_facts=None):
         # Store the timestamp once, then get local and UTC versions from that
         epoch_ts = time.time()
         now = datetime.datetime.fromtimestamp(epoch_ts)
-        utcnow = datetime.datetime.utcfromtimestamp(epoch_ts)
+        utcnow = utcfromtimestamp(epoch_ts).replace(tzinfo=None)
```

Comment:
Since `utcnow` is only used locally here to create a string without `tzinfo`, removing it is unnecessary:

```suggestion
        utcnow = utcfromtimestamp(epoch_ts)
```

Reply:
The initial change was not backwards incompatible, but I thought it was potentially less confusing in the future since `now` is naive - this keeps them the same type.